src/shared/hooks/useAccessDataForTable.tsx

```diff
@@ -422,43 +432,94 @@ const fileNameForDistributionItem = (distItem: any, defaultName: string) => {
   return fileName;
 };
 
-const resourceToLocalStorageResource = (resource: Resource): TDataSource => {
-  let localStorageRow: TDataSource;
+const getResourceName = (resource: Resource) =>
+  resource.name ?? resource['@id'] ?? resource._self;
+
+const baseLocalStorageObject = (
+  resource: Resource,
+  keySuffix?: string
+): Omit<TDataSource, 'distribution'> => {
+  return {
+    key: keySuffix ? `${resource._self}-${keySuffix}` : resource._self,
+    _self: resource._self,
+    id: resource['@id'],
+    name: getResourceName(resource),
+    project: resource._project,
+    description: resource.description ?? '',
+    createdAt: resource._createdAt,
+    updatedAt: resource._updatedAt,
+    source: 'studios',
+    type: isArray(resource['@type'])
+      ? resource['@type']
+      : [resource['@type'] ?? ''],
+  };
```

Comment:
why we take only the first one if the type is an array, 
I think the type is used to be showed in the data panel popup, so we can added all the types ?

Reply:
Hmm, I'm taking all the elements in the array if `resource[@type]` is an array. All these types are also shown in the datapanel:
![Screenshot from 2023-06-07 12-11-56](https://github.com/BlueBrain/nexus-web/assets/11242410/58f4d862-ccc0-4a54-affe-42db72d69ba2)